docs/dc-apps-performance-toolkit-user-guide-confluence.md

```diff
@@ -38,6 +38,24 @@ You are responsible for the cost of the AWS services used while running this Qui
 
 To reduce costs, we recommend you to keep your deployment up and running only during the performance runs.
 
+### AWS cost estimation ###
```

Comment:
[SIMPLE MONTHLY CALCULATOR](https://calculator.s3.amazonaws.com/index.html) provides an estimate of usage charges for AWS services based on certain information you provide.
Monthly charges will be based on your actual usage of AWS services, and may vary from the estimates the Calculator has provided.

_*The prices below are approximate and may vary depending on factors such as (region, instance type, deployment type of DB, etc.)_
| Confluence DC           | Costs ($/hr) |
| ------------------------| -------------|           
| One Node Confluence DC  | ≈ 1,5 - 1,6  |
| Two Node Confluence DC  | ≈ 1,7 - 2,5  |
| Four Node Confluence DC | ≈ 3,5 - 4,2  |

Reply:
Nice!
It seems good to me